src/main/java/dk/kb/datahandler/enrichment/FragmentsClient.java

```diff
@@ -0,0 +1,70 @@
+package dk.kb.datahandler.enrichment;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dk.kb.datahandler.config.ServiceConfig;
+import org.apache.http.client.utils.URIBuilder;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.*;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+public class FragmentsClient {
+
+    private static FragmentsClient instance;
+
+    private String baseUrl;
+
+    public static synchronized FragmentsClient getInstance() {
+        if (instance == null) {
+            instance = new FragmentsClient(ServiceConfig.getConfig().getString("fragmentService.baseUrl"));
+        }
+        return instance;
+    }
+
+    public FragmentsClient(String baseUrl) {
+        this.baseUrl = baseUrl;
+    }
+
+    public List<Fragment> fetchMetadataFragments(String id) throws IOException, URISyntaxException {
+        // Fetch enrichment data from the webservice
+        HttpURLConnection connection = getConnection(id);
+        if (connection.getResponseCode() != 200){
+            //do some error handling
+            return null;
+        }
+        BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream(), StandardCharsets.UTF_8));
+        String inputLine;
+        StringBuilder response = new StringBuilder();
+        while ((inputLine = in.readLine()) != null) {
+            response.append(inputLine);
+        }
+        in.close();
```

Comment:
These lines are duplicated in the DsPreservica client. Should we implement a client utils maybe?